module/VuFindSearch/src/VuFindSearch/Backend/SRU/Connector.php

```diff
@@ -246,7 +286,7 @@ protected function call($method = 'GET', $params = null, $process = true)
     protected function process($response)
     {
         // Send back either the raw XML or a SimpleXML object, as requested:
-        $result = XSLTProcessor::process('sru-convert.xsl', $response);
+        $result = XSLTProcessor::process('sru-convert-simple.xsl', $response);
```

Comment:
Should we consider deleting the original sru-convert.xsl file if it is no longer used anywhere? If there's no way to use it, even via configuration, it doesn't seem worth keeping around.

Reply:
Yes that's fair.  I think when I began this PR the WorldCat v1 code was still in there.  This is done.